packages/interbit-ui-tools/src/middleware/browserContext/index.js

```diff
@@ -3,8 +3,18 @@ const getHtmlConfig = require('./staticHtml')
 
 const getConfig = async () => getHtmlConfig(document)
 
+const getDefaultPort = () => {
```

Comment:
Please add source comments for all new/modified functions.

Reply:
This is not exposed in the public API and source comments are not applicable.